salt/docker/init.sls

```diff
@@ -24,6 +24,10 @@ include:
         {{ cert | indent(8) }}
     - require_in:
       - docker
+    module.run:
+      - name: service.systemctl_reload
+      - onchanges:
+        - file: /etc/docker/certs.d/{{ name }}/ca.crt
```

Comment:
This will run a `systemctl reload` for each cert. Not a good idea...

Reply:
> This will run a systemctl reload for each cert. Not a good idea...

we know that already from your previous comment, i just changed it to something that works until we had your feedback